.circleci/config.yml

```diff
@@ -11,6 +11,20 @@ install_and_start_kurtosis: &install_and_start_kurtosis
       kurtosis cluster set docker
       kurtosis engine start
 
+# Run quickstart (used in two places)
+run_quickstart: &run_quickstart
+  run: |
+    kurtosis run ./quickstart '[{"first_name":"Kevin", "last_name": "Bacon"}, {"first_name":"Steve", "last_name":"Buscemi"}]'
```

Comment:
i think the failure is coming from the sibling package "run ." workflow not being supported :-( 

@Peeeekay fysa

Reply:
oh yeah that's not supported; are we calling quickstart some other example package?

Reply:
yepp; that's not supported currently at least while running locally. Running it remotely should still work. In the pipeline, you could do something like `kurtosis run github.com/kurtosis-tech/awesome-kurtosis/quickstart@{CIRCLE_BRANCH}` should work.

Reply:
Echo'ing that @victorcolombo ran into this too